Algorithms/Common/DayofWeek.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+
+namespace Algorithms.Common
+{
+    public static class DayofWeek
+
+    {
+        /// <summary>
+        /// Determine day of week of the date
+        /// </summary>
+        /// <returns>Return string of day of week of the date</returns>
+        /// <param name="date">Date.</param>
+        public static string GetDayofWeek(DateTime date)
+        {
+
+            int count = 1; //Int to shows Ordinal number of occurrence of day of week
+            DateTime Occurrence = date.AddDays(-7); //Date to check the ordinal number of day of week 
+
+            for (string month = date.ToString("MMMM"); month == Occurrence.ToString("MMMM"); Occurrence = Occurrence.AddDays(-7))
+            {
+                count++;
+            }
+
+            //switch case for count to return string of day of week of the date
+            switch (count)
+            {
+                case 1:
+                    return count + "st " + date.ToString("ddd");
+                case 2:
+                    return count + "nd " + date.ToString("ddd");
+                case 3:
+                    return count + "rd " + date.ToString("ddd");
+                default:
+                    return count + "th " + date.ToString("ddd");
```

Comment:
1. "nd", "st", etc - without regional support.

> <returns>Return string of day of week of the date</returns>
2. Critical! You must return only day of week. But why you return week number too?
Or remove this switch, or change method description.
Want check weeks - do new method for SRP (SOLID).

You method it not count day of week. This only convert day number to string.